pkg/controller/wildflyserver/wildflyserver_controller.go

```diff
@@ -12,15 +12,19 @@ import (
 	wildflyutil "github.com/wildfly/wildfly-operator/pkg/controller/util"
 	"github.com/wildfly/wildfly-operator/pkg/resources"
 	"github.com/wildfly/wildfly-operator/pkg/resources/routes"
+	"github.com/wildfly/wildfly-operator/pkg/resources/servicemonitors"
 	"github.com/wildfly/wildfly-operator/pkg/resources/services"
 	"github.com/wildfly/wildfly-operator/pkg/resources/statefulsets"
 
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
```

Comment:
I do not know go language. But does this mean wildfly operator is relying on life resource? If github will have outage operator will not work properly?

Reply:
What do you mean by "life resources"?

This is equivalent to a Java package where we import the `monitoringv1` module from the GitHub repo when the application is built. It does not connect to github.com during the application runtime.